packaging/os/layman.py

```diff
@@ -248,7 +248,7 @@ def main():
         else:
             changed = uninstall_overlay(module, name)
 
-    except ModuleError, e:
+    except ModuleError as e:
```

Comment:
`as e` is valid when the module is already not py2.4 compatible for other reasons, normally external library dependency.